core/src/main/java/org/mskcc/cbio/portal/scripts/ImportFusionData.java

```diff
@@ -50,15 +50,18 @@
  */
 public class ImportFusionData {
     public static final String FUSION = "Fusion";
+    private final String GERMLINE = "GERMLINE";
     private File fusionFile;
     private int geneticProfileId;
     private String genePanelID;
+    private GeneticAlterationType geneticAlterationType;
     private Set<String> sampleSet  = new HashSet<>();
 
-    public ImportFusionData(File fusionFile, int geneticProfileId, String genePanelID) {
+    public ImportFusionData(File fusionFile, int geneticProfileId, String genePanelID, GeneticAlterationType geneticAlterationType) {
```

Comment:
I hope this does not break any other clients ... in particular, our pipeline importer code

Reply:
@sheridancbio shouldn't break our importer.. the import class is called with arguments 
```--data <datafile> --meta <metafile>``` so the way it's constructed here